projects/igniteui-angular/src/lib/chips/chip.spec.ts

```diff
@@ -172,36 +171,6 @@ describe('IgxChip', () => {
             expect(firstChipIconName).toContain('drag_indicator');
         });
 
-        it('should make chip comfortable when density is not set or it is set to comfortable', () => {
```

Comment:
As sizing is now just part of theming, I am not sure either. These tests make sense while the only way to set the size is an API call. When the comp has to react to sizes set at any of its parents, it seems like there is no single way to ensure it works anyway (i.e. the test could cause false security). We aren't testing if setting a red font color through element styles would make the font of the grid red right? I leaned towards removing these altogether because of this.

I'll discuss this with @simeonoff  and @damyanpetev